packages/config-array/src/config-array.js

```diff
@@ -307,6 +317,17 @@ function normalizeConfigPatterns(config) {
 
 	const newConfig = { ...config };
 
+	if (typeof config.basePath === "string") {
```

Comment:
Looks like we have this same check on line 297? Maybe an opportunity to store in a variable?

Reply:
Updated in https://github.com/eslint/rewrite/pull/223/commits/92a98b119b3cf27445bac8f1c53e52d70bb1e738